src/tooling/debugging/vscode-debugging.md

```diff
@@ -61,13 +61,13 @@ ESP32 doesn't have a built-in JTAG interface so you have to connect an external
 
 Older versions with **revision < 3** **don't** have built-in JTAG interface.
 
-ESP32-C3 with **revision 3** **does** have a built-in JTAG interface and you don't have to connect an external device to be able to debug. To get the chip revision, run the `cargo espflash board-info` command.
+ESP32-C3 with **revision 3** **does** have a built-in JTAG interface, and you don't have to connect an external device to be able to debug. To get the chip revision, run the `cargo espflash board-info` command.
 
 ### Hardware Setup
 
 If your ESP32-C3's revision is lesser than 3, follow these instructions, if you have revision 3 you can jump to the [**Set up VSCode**](#set-up-vscode-1) step.
 
-ESP32-C3 **revision 1** and **revision 2** don't have a built-in JTAG interface so you have to connect an external JTAG adapter to the ESP32-C3 board, for example, [ESP-Prog](https://docs.espressif.com/projects/espressif-esp-iot-solution/en/latest/hw-reference/ESP-Prog_guide.html) can be used.
+ESP32-C3 **revision 1** and **revision 2** don't have a built-in JTAG interface so, you have to connect an external JTAG adapter to the ESP32-C3 board, for example, [ESP-Prog](https://docs.espressif.com/projects/espressif-esp-iot-solution/en/latest/hw-reference/ESP-Prog_guide.html) can be used.
```

Comment:
```suggestion
ESP32-C3 **revision 1** and **revision 2** don't have a built-in JTAG interface, so you have to connect an external JTAG adapter to the ESP32-C3 board, for example, [ESP-Prog](https://docs.espressif.com/projects/espressif-esp-iot-solution/en/latest/hw-reference/ESP-Prog_guide.html) can be used.
```

Just a quick reference. This is a compound sentence with a coordinating conjunction (for, and, so, etc.). The comma should be placed before the coordinating conjunction. For details, see [this article](https://study.com/skill/learn/using-commas-with-compound-and-complex-sentences-explanation.html)